Element/CoordinatesUtility.php

```diff
@@ -100,9 +101,48 @@ public function render()
                 'MapbenderCoordinatesUtilityBundle:Element:coordinatesutility.html.twig',
                 [
                     'id'            => $this->getId(),
-                    'configuration' => $this->entity->getConfiguration(),
+                    'configuration' => $this->getConfiguration(),
                     'title'         => $this->getTitle()
                 ]
             );
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function getConfiguration()
+    {
+        $configuration = parent::getConfiguration();
+
+        if (isset($configuration['srsList']) && !empty($configuration['srsList'])) {
+            $configuration['srsList'] = $this->setSrsDefinitionsFromDatabase($configuration['srsList']);
+        }
+
+        return $configuration;
+    }
+
+
+    /**
+     * Get Srs definitions from database
+     *
+     * @param $srsList
+     * @return mixed
+     */
+    public function setSrsDefinitionsFromDatabase($srsList)
+    {
+        $repository = $repository = $this
```

Comment:
$repository = $this     
is better, think.

Reply:
omg :D